change/@fluentui-react-button-63f8f566-a6b7-4492-92ac-56510c1cd6df.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28616&commitId=829bb7fba472c8b41cf3fb5518b213bff0768a4b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Converged.Circular and icon only - RTL.pressed.chromium.png|97| Changed|
|Button Converged.Circular and icon only.hover.chromium.png|108| Changed|
|Button Converged.Circular and icon only.pressed.chromium.png|97| Changed|
|Button Converged.Circular and icon only - RTL.hover.chromium.png|108| Changed|
|Button Converged.Icon only.pressed.chromium.png|97| Changed|
|Button Converged.Icon only.hover.chromium.png|108| Changed|
|Button Converged.Size large.hover.chromium.png|152| Changed|
|Button Converged.Size small.pressed.chromium.png|97| Changed|
|Button Converged.Size large.pressed.chromium.png|136| Changed|
|Button Converged.Size small.hover.chromium.png|108| Changed|
|Button Converged.With icon before content - RTL.pressed.chromium.png|97| Changed|
|Button Converged.With icon before content.hover.chromium.png|108| Changed|
|Button Converged.With icon after content.hover.chromium.png|108| Changed|
|Button Converged.With icon before content.pressed.chromium.png|97| Changed|
|Button Converged.With icon before content - RTL.hover.chromium.png|108| Changed|
|Button Converged.With icon after content - RTL.hover.chromium.png|108| Changed|
|Button Converged.With icon after content - RTL.pressed.chromium.png|97| Changed|
|Button Converged.With icon after content.pressed.chromium.png|97| Changed|

</details>


  <details><summary><b>CompoundButton Converged</b> 14 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|CompoundButton Converged.Circular and icon only.hover.chromium.png|443| Changed|
|CompoundButton Converged.Circular and icon only - RTL.pressed.chromium.png|398| Changed|
|CompoundButton Converged.Circular and icon only.pressed.chromium.png|398| Changed|
|CompoundButton Converged.Icon only.pressed.chromium.png|398| Changed|
|CompoundButton Converged.Circular and icon only - RTL.hover.chromium.png|443| Changed|
|CompoundButton Converged.Size large.hover.chromium.png|443| Changed|
|CompoundButton Converged.Icon only.hover.chromium.png|443| Changed|
|CompoundButton Converged.Size small.hover.chromium.png|443| Changed|
|CompoundButton Converged.Size large.pressed.chromium.png|398| Changed|
|CompoundButton Converged.Size small.pressed.chromium.png|398| Changed|
|CompoundButton Converged.With icon after content.pressed.chromium.png|398| Changed|
|CompoundButton Converged.With icon before content - RTL.pressed.chromium.png|398| Changed|
|CompoundButton Converged.With icon after content - RTL.pressed.chromium.png|398| Changed|
|CompoundButton Converged.With icon before content.pressed.chromium.png|398| Changed|

</details>


  <details><summary><b>MenuButton Converged</b> 10 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MenuButton Converged.Circular and icon only.hover.chromium.png|108| Changed|
|MenuButton Converged.Icon only.hover.chromium.png|108| Changed|
|MenuButton Converged.Icon only.pressed.chromium.png|97| Changed|
|MenuButton Converged.Circular and icon only.pressed.chromium.png|97| Changed|
|MenuButton Converged.Size large.hover.chromium.png|152| Changed|
|MenuButton Converged.Size small.hover.chromium.png|108| Changed|
|MenuButton Converged.Size small.pressed.chromium.png|97| Changed|
|MenuButton Converged.Size large.pressed.chromium.png|136| Changed|
|MenuButton Converged.With icon.hover.chromium.png|108| Changed|
|MenuButton Converged.With icon.pressed.chromium.png|97| Changed|

</details>


  <details><summary><b>ToggleButton Converged</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ToggleButton Converged.Circular and icon only.hover.chromium.png|108| Changed|
|ToggleButton Converged.Circular and icon only.pressed.chromium.png|97| Changed|
|ToggleButton Converged.Icon only.hover.chromium.png|108| Changed|
|ToggleButton Converged.Icon only.pressed.chromium.png|97| Changed|
|ToggleButton Converged.Size large.pressed.chromium.png|136| Changed|
|ToggleButton Converged.Size large.hover.chromium.png|152| Changed|
|ToggleButton Converged.Size small.pressed.chromium.png|97| Changed|
|ToggleButton Converged.Size small.hover.chromium.png|108| Changed|
|ToggleButton Converged.With icon before content.pressed.chromium.png|97| Changed|
|ToggleButton Converged.With icon after content.hover.chromium.png|108| Changed|
|ToggleButton Converged.With icon after content.pressed.chromium.png|97| Changed|
|ToggleButton Converged.With icon before content.hover.chromium.png|108| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
The screenshot changes are expected.